src/main/resources/ai/privado/rulevalidator/schema/threats.json

```diff
@@ -0,0 +1,154 @@
+          "fix",
+          "repositories"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "$id": "#root/threats/items/id",
+            "title": "Id",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "Policy.Deny.Sharing.ContactToSlack"
+            ],
+            "pattern": "^.*$"
+          },
+          "name": {
+            "$id": "#root/threats/items/description",
+            "title": "Name",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "Policy to restrict Contact Information being shared with Slack"
+            ],
+            "pattern": "^.*$"
+          },
+          "description": {
+            "$id": "#root/threats/items/description",
+            "title": "Description",
+            "type": "string",
+            "default": "",
+            "examples": [
+              "Policy to restrict Contact Information being shared with Slack"
+            ],
+            "pattern": ".*"
+          },
+          "dataFlow": {
+            "$id": "#root/threats/items/dataFlow",
+            "title": "Dataflow",
+            "type": "object",
+            "required": [
+              "sources"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "sources": {
+                "$id": "#root/threats/items/dataFlow/sources",
+                "title": "Sources",
+                "type": "array",
+                "default": [],
+                "items":{
+                  "$id": "#root/threats/items/dataFlow/sources/items",
+                  "title": "Items",
+                  "type": "string",
+                  "format": "regex",
+                  "default": "",
+                  "examples": [
+                    "Data.Sensitive.ContactData.*"
+                  ],
+                  "pattern": "^.*$"
+                }
+              },
+              "sinks": {
+                "$id": "#root/threats/items/dataFlow/sinks",
+                "title": "Sinks",
+                "type": "array",
+                "default": [],
+                "items":{
+                  "$id": "#root/threats/items/dataFlow/sinks/items",
+                  "title": "Items",
+                  "type": "string",
+                  "format": "regex",
+                  "default": "",
+                  "examples": [
+                    "Data.Sharing.Slack"
+                  ],
+                  "pattern": "^.*$"
+                }
+              }
+            }
+          },
+          "repositories": {
+            "$id": "#root/threats/items/repositories",
+            "title": "Repositories",
+            "type": "array",
+            "default": [],
+            "items":{
+              "$id": "#root/threats/items/repositories/items",
+              "title": "Items",
+              "type": "string",
+              "format": "regex",
```

Comment:
This will fail if `**` is passed

Reply:
The format attr. is the type/medium we will be validating values with. Meaning we would use a regex to validate values against _repositories_. The type of value for this key is set to `string`, The schema version does support validation of values for being a valid regex though, but I didn't use it.